### PR TITLE
add all

### PR DESCRIPTION
**Pull request by oralag2**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+  margin: 5px 14px;
+  list-style: none;
+  font-weight: bold;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  color: black;
+  height: 60px;
```
Comment (IvanFesenko) [inline src/style.css:34]: Do not hard code height - use padding instead

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+
+.nav__item :hover{
+  color: blue;
+  border-bottom: 5px solid blue;
+  border-radius: 2px;
+
+
+}
+
+.LogoM{
```
Comment (IvanFesenko) [inline src/style.css:46]: fix class name
```suggestion
.logo{
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+  width: 60px;
+  height: 60px;
+
+}
+
+
+
+.example {
+  margin-top: 500px;
 }
```
Comment (IvanFesenko) [inline src/style.css:58]: Remove this

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Please provide demo link

**Comment by oralag2** [comment]
@IvanFesenko

**Comment by oralag2** [comment]
https://oralag2.github.io/layout_moyo-header/

**Pull request closed without merging**